tests/data/styles/scss/base/_code.scss

```diff
@@ -0,0 +1,4 @@
+code:not(pre code) {
+  background-color: black;
+  color: gray;
+}
```

Comment:
I added this in an attempt to reproduce an issue I noticed on CHQ. It did not work, but I’m keeping it because it doesn’t hurt to have a `:not()` selector in our test data.